bin/pkg.mli

```diff
@@ -1,3 +1,9 @@
 open Import
 
+module Lock : sig
+  (** [pp_packages lock_dir] returns a list of pretty-printed packages
+      occuring in [lock_dir]. *)
+  val pp_packages : Dune_pkg.Lock_dir.t -> 'a Pp.t list
```

Comment:
This function shouldn't exist in the public api.

Reply:
This isn't accessible since it's in the executable.

Reply:
Okay, well it's type should be `Pp.t` then.